tests/test_cli.py

```diff
@@ -575,3 +575,29 @@ def test_cli_finds_study_from_manifest_prefix(tmp_path):
     db = DuckDatabaseBackend(f"{tmp_path}/duck.db")
     tables = {x[0] for x in db.cursor().execute("show tables").fetchall()}
     assert "study_different_name__table" in tables
+
+
+@mock.patch.dict(os.environ, clear=True)
+@pytest.mark.parametrize(
+    "c_arg,raises",
+    [
+        ("foo:bar", does_not_raise()),
+        ("foo", pytest.raises(SystemExit)),
+    ],
+)
+def test_cli_custom_args(tmp_path, c_arg, raises):
```

Comment:
nit: would be nice to confirm the value gets parsed and sent down all the way to the study config too.

Reply:
fair point - i can mock something out